main/main.cpp

```diff
@@ -640,6 +642,130 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			print_line(get_full_version_string());
 			goto error;
 
+		} else if (I->get() == "--about") {
```

Comment:
I wonder if we could move this to another file like `main/show_license.h/cpp` or similar to avoid adding even more bloat to that file.

Reply:
I'll look into this. Should this be done with some static methods?